rest-api-spec/src/main/resources/rest-api-spec/test/cat.shards/10_basic.yml

```diff
@@ -1,7 +1,7 @@
 "Help":
   - skip:
       version: " - 2.11.99"
```

Comment:
@jed326 could you please bring these test version changes to `main` as well? Thank you